### DISCUSSION TITLE
Where to report data bugs?

### DISCUSSION THREAD

**Discussion by gsnedders** [ietf-tools/datatracker#4148] [Help / Questions] [answered]
It seems like the ietf-tools/datatracker repo is almost entirely about the software running the datatracker site; it's not clear however where one should file bugs relating to the data. Is this it? It would be good to have clarity there.

Specifically, in this case:

https://datatracker.ietf.org/doc/draft-tyoshino-hybi-permessage-compression/ should appear as an earlier version of https://datatracker.ietf.org/doc/rfc7692/.

**Comment by rjsparks** [comment, marked as answer]
To date, the same issue management system has been used for reporting both code and data issues. We'll evaluate that afresh, and provide clarity where we can.

For your particular data issue - the missing bit of data is the replaces relationship at draft-ietf-hybi-permessage-compression. The best thing to do when spotting such missing relationship (ietf doc replaces individual doc) is to send a note to the WG chair, ADs, or by default the IESG. A note to support@ietf.org would also work - the secretariat would dispatch it to the correct current leadership. I'll do that for you for this particular replaces.
